mtp_cashbook/templates/cashbook/transactions_history.html

```diff
@@ -40,18 +40,45 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">{% t
       </div>
     {% endif %}
 
-    {% for field in form %}
-      <div id="form-group-{{ field.html_name }}" class="form-group {% if field.errors %}error{% endif %}">
-        <label for="{{ field.id_for_label }}" id="{{ field.auto_id }}-label" class="form-label">
-          {{ field.label }}
-          {% for error in field.errors %}
-            <span class="error-message">{{ error }}</span>
-          {% endfor %}
-        </label>
-        {{ field }}
+    <div class="grid-row">
+    <div id="form-group-search" class="form-group ">
+        <div class="column-full">
+          <label for="id_search" id="id_search-label" class="form-label">
+            {% trans 'Prisoner name, prisoner number or sender name' %}
+            {% for error in field.errors %}
+              <span class="error-message">{{ error }}</span>
+            {% endfor %}
+          </label>
+          <input class=" form-control" id="id_search" name="search" type="text" />
+        </div>
       </div>
-    {% endfor %}
-
+    </div>
+
+    <div class="grid-row">
+      <div class="column-full"><label>Date received<label></div>
+      <div class="column-half">
+        <div id="form-group-start" class="form-group ">
+          <label for="id_start" id="id_start-label" class="form-label">
+            {% trans 'From' %}
+            {% for error in field.errors %}
+              <span class="error-message">{{ error }}</span>
+            {% endfor %}
+          </label>
```

Comment:
How will these `trans` blocks and `{{ error }}`s end up rendering? Should the `{{ errors }}` be interpolated into the translation, perhaps?